src/components/split-button/split-button.pw.tsx

```diff
@@ -0,0 +1,874 @@
+    mount,
+    page,
+  }) => {
+    await mount(
+      <SplitButton text="default text">
+        <Button href="#">Button 1</Button>
+        <Button>Button 2</Button>
+        <Button href="#">Button 3</Button>
+      </SplitButton>
+    );
+
+    await splitToggleButton(page).nth(0).click();
+    await expect(additionalButtonsContainer(page)).toHaveCSS(
+      "border-radius",
+      "8px"
+    );
+    await expect(
+      additionalButtonsContainer(page).locator("a").nth(0)
+    ).toHaveCSS("border-radius", "8px 8px 0px 0px");
+    await expect(
+      additionalButtonsContainer(page).locator("button").nth(0)
+    ).toHaveCSS("border-radius", "0px");
+    await expect(
+      additionalButtonsContainer(page).locator("a").nth(1)
+    ).toHaveCSS("border-radius", "0px 0px 8px 8px");
+  });
+
+  test(`should render with the expected border radius when there is only on one child button`, async ({
+    mount,
+    page,
+  }) => {
+    await mount(
+      <SplitButton text="default text">
+        <Button>Button 1</Button>
+      </SplitButton>
+    );
+
+    await splitToggleButton(page).nth(0).click();
+    await expect(additionalButton(page, 0)).toHaveCSS("border-radius", "8px");
+  });
+});
+
+test.describe("Prop tests", () => {
+  testData.forEach((text) => {
+    test(`should render with ${text} as text`, async ({ mount, page }) => {
+      await mount(<SplitButtonList text={text} />);
+
+      await expect(getDataElementByValue(page, "main-text")).toHaveText(text);
+    });
+  });
+
+  testData.forEach((subtext) => {
+    test(`should render with ${subtext} as subtext`, async ({
+      mount,
+      page,
+    }) => {
+      await mount(<SplitButtonList size="large" subtext={subtext} />);
+
+      await expect(buttonSubtextPreview(page)).toHaveText(subtext);
+    });
+  });
+
+  test(`should render with data-element prop`, async ({ mount, page }) => {
+    await mount(
+      <SplitButtonList data-element="split-button-cypress-element" />
+    );
+
+    await expect(splitMainButton(page)).toHaveAttribute(
+      "data-element",
+      "split-button-cypress-element"
+    );
+  });
+
+  test(`should render with data-role prop`, async ({ mount, page }) => {
+    await mount(<SplitButtonList data-role="split-button-cypress-role" />);
+
+    await expect(splitMainButton(page)).toHaveAttribute(
+      "data-role",
+      "split-button-cypress-role"
+    );
+  });
+
+  (["left", "right"] as SplitButtonProps["align"][]).forEach((alignment) => {
+    test(`should render with alignment to the ${alignment}`, async ({
+      mount,
+      page,
+    }) => {
+      await mount(<SplitButtonList align={alignment} />);
+
+      await expect(splitMainButton(page)).toHaveCSS(
+        `margin-${alignment}`,
+        "0px"
+      );
+    });
+  });
+
+  test(`should render with component disabled`, async ({ mount, page }) => {
+    await mount(<SplitButtonList disabled />);
+
+    await mainButton(page).hover();
+    await expect(
```

Comment:
**suggestion**: we should check that both buttons are disabled here, not just the first